linux/runtime.go

```diff
@@ -90,17 +95,18 @@ func New(ic *plugin.InitContext) (interface{}, error) {
 	}
 	cfg := ic.Config.(*Config)
 	r := &Runtime{
-		root:      ic.Root,
-		state:     ic.State,
-		remote:    !cfg.NoShim,
-		shim:      cfg.Shim,
-		shimDebug: cfg.ShimDebug,
-		runtime:   cfg.Runtime,
-		monitor:   monitor.(runtime.TaskMonitor),
-		tasks:     runtime.NewTaskList(),
-		db:        m.(*bolt.DB),
-		address:   ic.Address,
-		events:    ic.Events,
+		root:        ic.Root,
+		state:       ic.State,
+		remote:      !cfg.NoShim,
+		shim:        cfg.Shim,
+		shimDebug:   cfg.ShimDebug,
```

Comment:
can we put these in a struct together for the shim and runtime opts?

Reply:
What do you mean?

Reply:
Just so we don't have all these fields on the Runtime struct, just for grouping, not anything functional

Reply:
Oh, I see. 

Sure, will update